### ISSUE TITLE
"what ~/.pause is supposed to look like?"

### ISSUE THREAD

**Issue by zoffixznet** [skaji/mi6#44]
https://irclog.perlgeek.de/perl6/2018-04-25#i_16090469
```
14:04 	jkramer 	Zoffix: Any idea what ~/.pause is supposed to look like? Can't find docs for that
```

**Comment by skaji** [comment]
As you suggested, it looks like https://gist.github.com/zoffixznet/60452b30efe49e9b970412073b5605c3

**Comment by zoffixznet** [comment]
I wasn't clear in my bug report... Users are asking about it because it's not documented in the README.

If the module requires that files to exists, it should describe what the contents should look like.

**Comment by skaji** [comment]
Fixed by #55
